docs/docs/100-reference/01-command-line/acorn_logs.md

```diff
@@ -12,10 +12,11 @@ acorn logs [flags] [APP_NAME|CONTAINER_NAME]
 ### Options
 
 ```
-  -f, --follow         Follow log output
-  -h, --help           help for logs
-  -s, --since string   Show logs since timestamp (e.g. 42m for 42 minutes)
-  -n, --tail int       Number of lines in log output
+  -f, --follow                     Follow log output
+  -h, --help                       help for logs
+      --include-proxy-containers   Also display logs from Linkerd proxy containers
```

Comment:
This shouldn't be an option.  If someone want's to "break through" the abstraction acorn they can just use kubectl.  We don't want to expose the internal details of the platform to the user.

Reply:
Removed. Through from playing around, I did figure out it is still technically possible for them to get the linkerd logs through acorn by running something like `acorn logs <app name>.<container name>.linkerd-proxy`. I could look for this case and explicitly block it if we want, but I don't know that that is worth doing.

Reply:
That's probably fine.